news/_posts/translation/2020-07-10-do-you-speak-prestashop-june-2020.md

```diff
@@ -0,0 +1,126 @@
+
+## A few stats
+
+* **27 members** were active on the project this month.
+* A total of **18,283 words have been translated** and **7,841 validated**.
+* All this in **21 different languages**.
+
+Thank you for your involvement!
+
+
+## Top contributors
+
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and we can't thank you enough for your dedication! Here are the **most active translators and proofreaders** for May 2020.
+
+#### Top 10 translators in June:
+
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+ 1. | Rolige Ecommerce Solutions (Rolige) | Spanish, Mexico | 4,827
+ 2. | mirmal | Esperanto | 4,500
+ 3. | Gerardas (gerardas) | Lithuanian | 1,594
+ 4. | kozi lamre (kozmetika2000) | Albanian | 1,454
+ 5. | Yevhenii (evgenij_sobolev) | Russian | 1,334
+ 6. | Gabriel Tenita (ggedamed) | Romanian | 1,170
+ 7. | Maciej Wyka (BeroMero) | Polish | 939
+ 8. | Rauno Riikman (weaver) | Finnish | 893
+ 9. | Waleed Butt (waleedbutt112) | Urdu (Pakistan) | 363
+10. | Daniel Hlavacek (Hlavtox) | Czech | 202
+
+
+#### Top 7 proofreaders in June:
+
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+ 1. | Rauno Riikman (weaver) | Finnish | 3,226
+ 2. | Gerardas (gerardas) | Lithuanian | 1,735
+ 3. | Yevhenii (evgenij_sobolev) | Russian | 1,643
+ 4. | Gabriel Tenita (ggedamed) | Romanian | 1,203
+ 5. | Zoran Tejic (zotamal) | Serbian (Latin) | 26
+ 6. | Marcin Orzechowski (Martinovy) | Polish | 5
+ 7. | Rodrigo Laurindo (rblaurin) | Portuguese, Brazilian | 3
+ 
+Congrats, and welcome to our new top contributors! :clap:
+
+Remember, you can see who's been contributing to our translation project thanks to the [Translators page](http://translators.prestashop.com/).
+
+
+## Complete translations
+
+#### Fully translated languages
+
+At the end of June 2020, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **10 languages**:
+
+| Chinese Traditional | Danish | Dutch | French | Italian |
+| Portuguese, Brazil | Slovak | Slovene | Spanish | Swedish |  
+
+
+## Languages with the best evolution
+
+In comparison with May 2020, the following languages had the best progress thanks to the translation community:
+
+* **Esperanto** (+9% to reach 34% translated)
+* **Finnish** (+6% to reach 92% approved)
+* **Spanish, Mexico** (+5% to reach 63% translated)
+
+![Best translation progress for June 2020](/assets/images/2020/06/build-crowdin-progress-june20.jpg)
```

Comment:
```suggestion
![Best translation progress for June 2020](/assets/images/2020/07/build-crowdin-progress-june20.png)
```
take great care with picture extensions and links. we spoke already about that, jpg is not the same than png :-)